Source/ProgressionSystemRuntime/ProgressionSystemRuntime.Build.cs

```diff
@@ -11,10 +11,10 @@ public ProgressionSystemRuntime(ReadOnlyTargetRules Target) : base(Target)
 		PCHUsage = PCHUsageMode.UseExplicitOrSharedPCHs;
 		PublicDependencyModuleNames.AddRange(new[]
 			{
-				"Core", "UMG"
+				"Core", "UMG" 
+				, "GameFeatures"
 				// Bomber modules
 				, "Bomber" // Is included in header files
-				, "SettingsWidgetConstructor" // SettingWidgetConstructor
```

Comment:
You just removed important `SettingsWidgetConstructor` dependency while resolving merge conflict. You can notice it was present on left side, but it's gone now on the right side. Because of this, your brunch does not compile anymore:
```
0>PSDataAsset.h(5): Error C1083 : Cannot open include file: 'Data/SettingTag.h': No such file or directory
```

Please always recompile and test after each merge conflict

Reply:
reverted with: https://github.com/h4rdmol/ProgressionSystem/pull/26/commits/09bfccd83e624edaaea9d505ebdd5e754500b050

Reply:
@h4rdmol, Please follow formatting: start new line with comma (,) character:
```
// Bomber modules
, "Bomber"
, "SettingsWidgetConstructor"

Reply:
done https://github.com/h4rdmol/ProgressionSystem/pull/26/commits/a7207386e5c82c9582077a3c395dd97d9ae60f9a